tools/autograd/derivatives.yaml

```diff
@@ -1662,22 +1662,22 @@
 - name: _to_dense(Tensor self, ScalarType? dtype=None) -> Tensor
   self: to_dense_backward(grad, self)
 
-- name: to_sparse(Tensor self, *, Layout? layout=None, int[2]? blocksize=None) -> Tensor
+- name: to_sparse(Tensor self, *, Layout? layout=None, int[2]? blocksize=None, int? n_dense_dim=None) -> Tensor
   self: to_sparse_backward(grad, self.layout(), self.sym_blocksize())
```

Comment:
Do we have tests for these derivatives anywhere? I think the `n_dense_dim` arg will have to be stashed for backward as well.

Reply:
It's not necessary to use `n_dense_dim` for derivatives for now.  Namely, in the forward direction it's only used for dense to compressed sparse (i.e. to CSR/CSC/BSR/BSC) conversions.  In the backward direction, it is conversion to dense, and `n_dense_dim` is not needed there.